terraform/user.tf

```diff
@@ -10,8 +10,29 @@ module "user" {
     aws.images-staging-ssm                 = aws.images_staging_ssm
   }
 
-  entity         = "ansible-role-venom-nessus-agent"
-  ssm_parameters = ["/github/personal_authorization_token"]
+  entity = "ansible-role-venom-nessus-agent"
+  # The TF module will error if we don't put at least one value here.
+  # This build user does not need to access any SSM parameters, so we
+  # just place a dummy value here.
+  ssm_parameters = ["/dummy/value"]
```

Comment:
I think there's something I'm missing, but is there a reason we can't use [cisagov/ci-iam-user-tf-module](https://github.com/cisagov/ci-iam-user-tf-module) to avoid needing a dummy value here?

Reply:
Fixed in commit b2622fd.